react/Table/Virtualized/index.jsx

```diff
@@ -113,14 +115,32 @@ VirtualizedTable.defaultProps = {
   selectedItems: [],
   isSelectedItem: () => {},
   onSelect: () => {},
-  onSelectAll: () => {}
+  onSelectAll: () => {},
+  onSortChange: () => {}
 }
 
 VirtualizedTable.propTypes = {
-  /** Column ID to be used for initiating the sort */
-  defaultOrder: PropTypes.string,
+  /** Rows to display in the table */
+  rows: PropTypes.array,
+  /** Column configuration */
+  columns: PropTypes.array,
   /** Returned object is: PropTypes.shape({ groupLabels: PropTypes.array, groupCounts: PropTypes.array }) */
   groups: PropTypes.func,
+  /** Default sorting configuration */
+  defaultOrder: PropTypes.shape({
+    direction: PropTypes.oneOf(['asc', 'desc']),
+    by: PropTypes.string
+  }),
+  /** Array of secondary sort configurations */
+  secondarySort: PropTypes.array,
+  /** Array of selected items */
+  selectedItems: PropTypes.array,
+  /** Callback function when a row is selected */
+  onSelect: PropTypes.func,
+  /** Callback function when all rows are selected/deselected */
+  onSelectAll: PropTypes.func,
```

Comment:
it's a func triggered only when click "all select" checked. The fact that it's a toggle to select/deselect is handle by the app, not here I think (and not really a callback actually I think, like `onSelect`)